.github/actions/run-interop-ping-test/action.yml

```diff
@@ -9,9 +9,40 @@ inputs:
     description: "Space-separated paths to JSON files describing additional images"
     required: false
     default: ""
+  s3-cache-bucket:
+    description: "Which S3 bucket to use for container layer caching"
+    required: false
+    default: ""
+  s3-access-key-id:
+    description: "S3 Access key id for the cache"
+    required: false
+    default: ""
+  s3-secret-access-key:
+    description: "S3 secret key id for the cache"
+    required: false
+    default: ""
+  aws-region:
+    description: "Which AWS region to use"
+    required: false
+    default: "us-east-1"
```

Comment:
Can these be env variables so we don't need to pass them on explicitly?

Reply:
I picked this since it's explicit this is an input to the test. Env variables aren't as explicit.

Reply:
I see. Perhaps explicit is better, 4 inputs just felt like a lot of clutter.

Reply:
That's how much is needed for the s3 cache to work. It'll test without them of course.